lib/tasks/backfill_probation_service.rake

```diff
@@ -5,8 +5,7 @@ require 'rake'
 namespace :backfill do
   desc 'Back-fill case_information#probation_service from welsh_offender'
   task probation_service: :environment do
-    CaseInformation.where(probation_service: nil).find_each do |ci|
-      ci.update!(probation_service: (ci.welsh_offender == 'Yes') ? 'Wales' : 'England')
-    end
+    CaseInformation.where(welsh_offender: 'Yes').update_all(probation_service: 'Wales')
```

Comment:
needs to be probation_service: nil as well

Reply:
@starswan done